filelogger_test.go

```diff
@@ -2,7 +2,7 @@ package goolog2_test
 
 import (
 	"bytes"
-	. "goolog2"
+	. "github.com/Staon/goolog2"
```

Comment:
Předpokládám, že tohle vynutily ty nové moduly?

Reply:
Jo. Zrovna u toho jsem si říkal, jestli bych to neměl rozdělit na dva kommity (moduly + rotate). Nakonec jsem se na to vykašlal.